src/server/api.ts

```diff
@@ -145,6 +132,21 @@ router.get("/login/success", async (req, res) => {
   return res.redirect(state);
 });
 
+router.post("/nodebb-cookie", async (req, res) => {
```

Comment:
Kan ikke hvem som helst kalle på dette endepunktet uten å ha noen tilknytning til nodebb?

Reply:
Jo i prinsippet. Har du et bedre forslag til korleis sette cookie kun for pålogga med tilgang?

Reply:
Sender ikke NodeBB med noe unikt som vi kan validere på en eller annen måte?

Reply:
Poenget her er å lage en cookie som nodebb skal sjekke. Denne kalles fra AuthenticationContext.

Reply:
Betyr det at hvem som helst kan logge inn så lenge de har en feideid?
Det høres jo ganske uheldig ut?

Reply:
Arena skal åpnes for alle  bortsett fra elever om ikkje alt for lenge, så det er ikkje veldig alvorlig.

Reply:
Bare sånn for completeness: hvem som helst hadde kunt logge inn som hvem som helst andre. Så det hadde vært litt alvorlig 😄